e2e-tests/tests/420-text-plugin.ts

```diff
@@ -20,24 +20,11 @@ function focusTextPlugin(I, position = 'first') {
   }
 }
 
-Scenario('Add a new line using Enter', async ({ I }) => {
-  I.amOnPage(pageUrl)
-  I.seeNumberOfElements(selectors.textEditor, initialTextPluginCount)
-
-  addNewTextPlugin(I)
-  I.seeNumberOfElements(selectors.textEditor, initialTextPluginCount + 1)
-
-  I.say('Press Enter to add a new line')
-  I.pressKey('Enter')
-  I.seeNumberOfElements(selectors.textEditor, initialTextPluginCount + 1)
-})
-
 Scenario('Add new line in plugin using Enter', async ({ I }) => {
   I.amOnPage(pageUrl)
   I.seeNumberOfElements(selectors.textEditor, initialTextPluginCount)
 
-  addNewTextPlugin(I)
-  I.seeNumberOfElements(selectors.textEditor, initialTextPluginCount + 1)
+  I.click(locate(selectors.textEditor).inside('.plugin-rows'))
```

Comment:
I don't mind tests to be verbose. In this case you could consider maybe moving this to a helper. This way it would also be easier to read (was not sure that this does at first).

Reply:
I added `I.say` statements to improve readability.